packages/next/src/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -99,7 +99,7 @@ interface UninlinedCssFile {
 export interface ClientReferenceManifest extends ClientReferenceManifestForRsc {
   readonly moduleLoading: {
     prefix: string
-    crossOrigin: string | null
+    crossOrigin?: 'use-credentials' | ''
```

Comment:
I prefer to have explicit type even for undefined rather than optional for manifest.
```suggestion
    crossOrigin: 'use-credentials' | '' | undefined
```

Reply:
This just mirrors the types that React has defined: https://github.com/facebook/react/blob/040f8286e97eab7aac23a89d5f59967be3bf8f3e/packages/react-server-dom-webpack/src/client/ReactFlightClientConfigTargetWebpackServer.js#L12-L15